docs/DafnyRef/Attributes.md

```diff
@@ -141,32 +95,96 @@ be added.
    if (F != null) { Repr := Repr + {F} + F.Repr; }
 ```
 
-### 22.1.4. axiom
+### 22.1.2. `{:nativeType}` {#sec-nativetype}
+The `{:nativeType}` attribute may only be used on a ``NewtypeDecl``
+where the base type is an integral type. It can take one of the following
+forms:
+
+* `{:nativeType}` - With no parameters it has no effect and the ``NewtypeDecl``
+have its default behavior which is to choose a native type that can hold any
+value satisfying the constraints, if possible, otherwise BigInteger is used.
+* `{:nativeType true}` - Also gives default ``NewtypeDecl`` behavior,
+but gives an error if base type is not integral.
+* `{:nativeType false}` - Inhibits using a native type. BigInteger is used
+for integral types and BitRational for real types.
```

Comment:
I suggest leaving out "and BitRational for real types", since the text above says `:nativeType` applies only to integral types. (But if these words are kept, then "BitRational" should be corrected to "BigRational".)